src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -8276,6 +8276,13 @@ public sealed partial class ExcludeFromCodeCoverageAttribute : System.Attribute
         public ExcludeFromCodeCoverageAttribute() { }
         public string? Justification { get { throw null; } set { } }
     }
+    [System.AttributeUsage(System.AttributeTargets.Assembly |  System.AttributeTargets.Module | System.AttributeTargets.Class | System.AttributeTargets.Struct | System.AttributeTargets.Enum | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Property | System.AttributeTargets.Field | System.AttributeTargets.Event | System.AttributeTargets.Interface | System.AttributeTargets.Delegate, Inherited = false)]
+    public sealed class ExperimentalAttribute : Attribute
+    {
+        public ExperimentalAttribute(string diagnosticId) { }
+        public string DiagnosticId { get { throw null; } }
+        public string? UrlFormat { get; set; }
+    }
```

Comment:
I haven't found any docs whether this file is modified manually or by some tool... So I manually edited it.

Reply:
There is a command to do it, but manual editing is ok. It just increases the liklihood of more diffs the next time someone runs the tool.

Reply:
Can you please point me to the doc describing the command?

Reply:
https://github.com/dotnet/runtime/blob/bd7e1cb81d763589b95f7bfbaa2a96dd364e9103/docs/coding-guidelines/updating-ref-source.md

Reply:
Note that this case we are adding it to System.Runtime, so you will need to follow this section: https://github.com/dotnet/runtime/blob/bd7e1cb81d763589b95f7bfbaa2a96dd364e9103/docs/coding-guidelines/updating-ref-source.md#for-systemruntime

Reply:
Awesome, thank you!